imesec-full-stack.yml

```diff
@@ -0,0 +1,91 @@
+version: "3.4"
+services:
+  lh-web:
+    image: razgrizone/lightning-hacks-website
+    environment:
+      VIRTUAL_HOST: lh.torradeira.imesec.ime.usp.br
+    deploy:
+      replicas: 2
+      restart_policy:
+        condition: any
+        delay: "0"
+    ports:
+      - "8994:5000"

```

Comment:
Since we'll have the automagic proxy, do we need these ports to be open as well?